extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -940,10 +952,18 @@ export class AmpA4A extends AMP.BaseElement {
             dev().error(TAG, this.element.getAttribute('type'),
                 'Error executing onAmpCreativeRender', err);
           })();
+          // Capture timing info for friendly iframe load completion.
+          getTimingDataAsync(friendlyIframeEmbed.win,
+              'navigationStart', 'loadEventEnd').then(delta => {
+                this.emitLifecycleEvent('renderFriendlyEnd', {
```

Comment:
I was thinking we would have a new event (e.g. load) which we could apply to both friend and non-friend render flows.